src/libPMacc/PMaccConfig.cmake

```diff
@@ -66,11 +66,19 @@ if(NOT "${CMAKE_CXX_FLAGS}" MATCHES "-std=c\\+\\+11")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
 endif()
 
-set(CUDA_ARCH sm_20 CACHE STRING "Set GPU architecture")
-string(COMPARE EQUAL ${CUDA_ARCH} "sm_10" IS_CUDA_ARCH_UNSUPPORTED)
-string(COMPARE EQUAL ${CUDA_ARCH} "sm_11" IS_CUDA_ARCH_UNSUPPORTED)
-string(COMPARE EQUAL ${CUDA_ARCH} "sm_12" IS_CUDA_ARCH_UNSUPPORTED)
-string(COMPARE EQUAL ${CUDA_ARCH} "sm_13" IS_CUDA_ARCH_UNSUPPORTED)
+set(CUDA_ARCH 20 CACHE STRING "Set GPU architecture (semicolon or comma separated list, e.g. '-DCUDA_ARCH=20,35,60')")
+
+# convert all commas to semicolons to support the following foreach call
+string(REPLACE "," ";" PMACC_CUDA_ARCH ${CUDA_ARCH})
```

Comment:
I would make it a semicolon separeted list by requirement and spare all the inconveniences and corner cases that come in when allowing both. There are many scripts setting and using this and allowing two syntax forms just causes trouble because all of them have to care.